chain/penumbra/penumbra_chain.go

```diff
@@ -140,6 +140,10 @@ func (c *PenumbraChain) CreateKey(ctx context.Context, keyName string) error {
 	return c.getRelayerNode().PenumbraAppNode.CreateKey(ctx, keyName)
 }
 
+func (c *PenumbraChain) RecoverKey(ctx context.Context, name, mnemonic string) error {
+	return fmt.Errorf("RecoverKey not implemented for PenumbraChain")
+}
```

Comment:
I'm not sure exactly what would need to be done here.

Reply:
I think this error is fine for now.